draft-wkumari-dhc-addr-notification.md

```diff
@@ -204,26 +208,27 @@ The ADDR-REG-INFORM message MUST contain an IA Address option for the address be
 
 Servers MUST ignore any received ADDR-REG-REPLY messages.
 
-The IPv6 destination address of the packet is the address being registered.
+Clients MUST discard any ADDR-REG-REPLY messages that meet any of the following conditions:
+- the IA Address option does not contain the address assigned on the interface the client received the message on.
```

Comment:
I see you deleted the text that says the destination address of the ADDR-REG-REPLY is the address being registered. Is that intentional? If not, maybe we can put it back, and make this text say:
- The IPv6 destination address does not match the address being registered
- The IA-Address option does not match the address being registered
- The address being registered is not assigned to the interface receiving the message.

Reply:
I've updated the text